libs-scala/contextualized-logging/src/main/scala/com/digitalasset/logging/LoggingContext.scala

```diff
@@ -9,22 +9,23 @@ import org.slf4j.Marker
 
 import scala.collection.JavaConverters.mapAsJavaMapConverter
 
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
 object LoggingContext {
 
   def newLoggingContext[A](f: LoggingContext => A): A =
     f(new LoggingContext(Map.empty))
 
-  def newLoggingContext[A](kv: (String, String), kvs: (String, String)*)(
-      f: LoggingContext => A): A =
+  def newLoggingContext[A](kv: (String, Any), kvs: (String, Any)*)(f: LoggingContext => A): A =
     f(new LoggingContext((kv +: kvs).toMap))
 
-  def withEnrichedLoggingContext[A](kv: (String, String), kvs: (String, String)*)(
-      f: LoggingContext => A)(implicit logCtx: LoggingContext): A =
+  def withEnrichedLoggingContext[A](kv: (String, Any), kvs: (String, Any)*)(f: LoggingContext => A)(
+      implicit logCtx: LoggingContext): A =
     f((logCtx + kv) ++ kvs)
 
 }
 
-final class LoggingContext private (ctxMap: Map[String, String]) {
+@SuppressWarnings(Array("org.wartremover.warts.Any"))
```

Comment:
Do you need to put this around the whole class? Can you not restrict the scope of this and possibly annotate the methods and the calls explicitly to remove this?